DataFormats/MuonReco/src/classes_def.xml

```diff
@@ -65,7 +65,8 @@ initial version number of a class which has never been stored before.
    <version ClassVersion="11" checksum="541727491"/>
    <version ClassVersion="10" checksum="4050071853"/>
   </class>
-  <class name="reco::MuonSegmentMatch" ClassVersion="11">
+  <class name="reco::MuonSegmentMatch" ClassVersion="12">
+    <version ClassVersion="12" checksum="3329152288"/>
```

Comment:
I'd expect the `MuonSegmentMatch` member muon hit Refs to be broken here as well.